src/node_file.cc

```diff
@@ -2727,6 +2730,222 @@ static void Mkdtemp(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static bool FileURLToPathImpl(Environment* env,
+                       ada::url_aggregator* file_url,
+                       std::unique_ptr<std::string>& result_file_path) {
```

Comment:
I don't understand why we don't do this?

```suggestion
                       std::string& result_file_path) {
```

Reply:
Fixed!

Reply:
But this causes me a lint issue:

```
Running C++ linter...
/home/h4ad/Projects/opensource/node/tools/cpplint.py:57: DeprecationWarning: module 'sre_compile' is deprecated
  import sre_compile
src/node_file.cc:2735:  Is this a non-const reference? If so, make const or use a pointer: std::string& result_file_path  [runtime/references] [2]
Done processing src/node_file.cc
```

If I make it const, I will not be able to modify it, right?